src/components/common/Extensions/UserHub/partials/ReputationTab/partials/Balance.tsx

```diff
@@ -60,7 +60,7 @@ const Balance: FC<BalanceProps> = ({ nativeToken, wallet, onTabChange }) => {
 
   return (
     <div>
-      <TitleLabel text={formatMessage({ id: 'balance.in.colony' })} />
+      <TitleLabel text={formatMessage({ id: 'userHub.nativeToken' })} />
```

Comment:
Thank you for this. In the future, I'd be happy if we moved this kind of stuff back into the individual components using `defineMessages`